OrbitGl/App.cpp

```diff
@@ -704,6 +716,7 @@ void OrbitApp::ClearCapture() {
 
   SetSamplingReport(empty_profiler, empty_unique_callstacks);
   SetTopDownView(GetCaptureData());
+  SetSelectionTopDownView(empty_profiler, GetCaptureData());
```

Comment:
I saw, that you were grouping the update of the sampling related views below, maybe this should be also done here?

Reply:
I introduced `UpdateAfterCaptureCleared`, please have a look.